common/errors/errors_test.go

```diff
@@ -0,0 +1,83 @@
+package errors_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/pastelnetwork/gonode/common/errors"
+)
+
+func TestNew(t *testing.T) {
```

Comment:
I see, you took it from `github.com/go-errors/errors/error_test.go` what's the point?

Reply:
my idea is a code with test is better than one without. this can def. be improved this is mainly coverage not 100% functionality so more test cases can be added to expose errors package intention.

Reply:
I cannot agree with your opinion, because if have `_test.go` no one else will care about this unit test, all guys will think _this's already done_ and no one even thinks to improve it. I've known a lot of guys who like refactoring code, but I've never met anyone who would like to refactor a unit test.
So, I believe, if we write unit test it should really test our package, rather than imitation. If we don't have time for unit testing yet, it's best not to write anything. In this case, we will always remember this as what we must do. 
And of course, we should not throw words to the wind, plan but not do.

https://blockchain-devel.slack.com/archives/C01SLR99SAK/p1620239309035100?thread_ts=1620221825.025900&cid=C01SLR99SAK